whois-api/src/test/java/net/ripe/db/whois/api/elasticsearch/ElasticAutocompleteServiceTestIntegration.java

```diff
@@ -67,12 +67,12 @@ public void setUp() throws Exception {
     }
 
     @Override
-    String getWhoisIndex() {
+    public String getWhoisIndex() {
```

Comment:
Do we need the `public` modifier?

Reply:
Yes autocomplete and fulltext integration tests are in separate packages and both needs to be override this method